packages/@sanity/base/src/_exports/hooks.ts

```diff
@@ -1,4 +1,8 @@
-export {useDocumentPresence, useGlobalPresence} from '../datastores/presence/hooks'
+export {
+  useDocumentPresence,
+  useGlobalPresence,
+  useGlobalDocumentPresence,
+} from '../datastores/presence/hooks'
```

Comment:
What is the difference between `useDocumentPresence` and `useGlobalDocumentPresence`?

Reply:
- `useDocumentPresence(id: string)` takes a document id and returns the presence for it
- `useGlobalPresence()` takes no arguments and returns an array of presence items
- `useGlobalDocumentPresence()` is a variant of `useGlobalPresence` that returns a record of `documentId=>PresenceItem[]`.